fineract-provider/dev-dependencies.gradle

```diff
@@ -92,17 +89,19 @@ dependencies {
                 //[group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.0']
 
                 [group: 'io.swagger', name: 'swagger-jersey-jaxrs', version: '1.5.15'],
-                [group: 'org.springframework', name:'spring-jms'],
-                [group: 'org.apache.activemq', name: 'activemq-broker'],
+                [group: 'org.springframework', name:'spring-jms', version: '4.0.7.RELEASE'],
```

Comment:
It is really nice idea. But I would propose to move it into dedicated task, just to keep the current task shorter.

Reply:
Of course, I didn't mean to do that in this PR, but later; FYI I've just created https://issues.apache.org/jira/browse/FINERACT-797 re. this.. just in case you'd like to pick that up some day! :smiling_imp: